zeus/device/cpu/rapl.py

```diff
@@ -86,6 +177,10 @@ def __init__(self, path: str) -> None:
                 "Error reading package max energy range"
             ) from err
 
+        self.wraparound_tracker = RaplWraparoundTracker(
+            self.energy_uj_path, self.max_energy_range_uj
+        )
+
     def __str__(self) -> str:
```

Comment:
It's usually better if `__str__` methods include the name of the class (`RAPLFile`) in their string representation.